azure-pipelines.yml

```diff
@@ -80,6 +80,15 @@ jobs:
     condition: eq(variables['Build.Reason'], 'PullRequest')
     displayName: 'Use clang-format to check C/C++ coding style'
 
+  - bash: |
+      if [[ -z "$(find sw -name '*.h' -type f -print0 | xargs -0 util/fix_include_guard.py --dry-run | tee /dev/stderr)" ]]; then
```

Comment:
Shouldn't we filter out sw/vendor?

Reply:
The script already ignores all paths with 'vendor' in them, or paths not ending in '.h'.